libraries/hoel/hoel.info

```diff
@@ -1,7 +1,7 @@
 PRGNAM="hoel"
-VERSION="1.4.22"
+VERSION="1.4.23"
 HOMEPAGE="https://github.com/babelouest/hoel"
-DOWNLOAD="https://github.com/babelouest/hoel/archive/v1.4.22/hoel-1.4.22.tar.gz"
+DOWNLOAD="https://github.com/babelouest/hoel/archive/v1.4.23/hoel-1.4.23.tar.gz"
 MD5SUM="e5d9bd2dfb35c8fc4435f51caa21ae9b"
```

Comment:
MD5SUM does not match

Reply:
You're right, I've broken my script, it is fixed, sorry.